RELEASE.md

```diff
@@ -49,9 +49,9 @@ yarn install
 
 * And last (but not least 😁) do your release.
 
-```sh
-npx release-it
```

Comment:
Hmm, using `npx release-it` does exactly what you are asking (in this PR). If `release-it` is installed in the repo, it will use the local bin otherwise it will install it in a temp dir and use it.